data.js

```diff
@@ -1,4 +1,5 @@
 import emojiData from 'emoji-datasource';
+import "babel-polyfill";
```

Comment:
they suggest here https://babeljs.io/docs/usage/polyfill/ to put it on the entry point.

Reply:
yep, but in that case, you should patch  project... 
I added fix with those you can just install lib and use it

Reply:
I meant entry point of the lib - https://github.com/goshakkk/react-native-simple-emoji-picker/blob/master/index.js

Reply:
I tried to include in index.js - I got an error